### PR TITLE
Updated French translation.

### PR DESCRIPTION
**Pull request by LapinFou**
Updated French translation once again... 😉

### CODE REVIEW

File: companion/src/translations/companion_fr.ts [hunk 1]
```diff
@@ -4340,265 +4320,315 @@ Nous recommandons de lire attentivement les notes de version à l&apos;aide du b
...
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1469"/>
+        <source>Confirm Delete Profile</source>
+        <translation>Confirmer la suppression du profil</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1470"/>
+        <source>Are you sure you wish to delete the &quot;%1&quot; radio profile? There is no way to undo this action!</source>
+        <translation>Etes-vous sûr de vouloir effacer le profil radio %1 ? C&apos;est action est irréversible !</translation>
```
Comment (bsongis) [inline companion/src/translations/companion_fr.ts]: Cette

File: companion/src/translations/companion_fr.ts [hunk 2]
```diff
@@ -4340,265 +4320,315 @@ Nous recommandons de lire attentivement les notes de version à l&apos;aide du b
...
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1090"/>
+        <source>Delete Current Radio Profile</source>
+        <translation>Effacer le profil radio actuel</translation>
+    </message>
+    <message>
+        <location filename="../mainwindow.cpp" line="1090"/>
+        <source>Delete the current Radio Settings Profile</source>
+        <translation>Affacer les réglages radio du profil actuel</translation>
```
Comment (bsongis) [inline companion/src/translations/companion_fr.ts]: Effacer

File: companion/src/translations/companion_fr.ts [hunk 3]
```diff
@@ -9328,420 +9370,205 @@ Si cette option est cochée, la voie des gaz est inversée: le ralenti  est &apo
...
+        <location filename="../simulation/simulator_strings.h" line="27"/>
+        <source>&lt;font size=+3&gt;&amp;#x21d1;&lt;/font&gt;</source>
         <translation></translation>
     </message>
     <message>
-        <location filename="../simulation/simulateduiwidgetX9.cpp" line="48"/>
-        <source>-/DN</source>
-        <translation>-/BAS</translation>
+        <location filename="../simulation/simulator_strings.h" line="29"/>
+        <source>&lt;font size=+3&gt;&amp;#x21d5;&lt;/font&gt;</source>
```
Comment (bsongis) [inline companion/src/translations/companion_fr.ts]: Those strings should not be translatable!

Comment (LapinFou) [inline companion/src/translations/companion_fr.ts]: Do you mean all strings in the `SimulatedUIWidget`?

File: companion/src/translations/companion_fr.ts (hunk 2 above)
Comment (LapinFou) [inline companion/src/translations/companion_fr.ts]: Oups !! Done.

File: companion/src/translations/companion_fr.ts (hunk 1 above)
Comment (LapinFou) [inline companion/src/translations/companion_fr.ts]: Done.

**Comment by LapinFou** [comment]
Updated done.

File: companion/src/translations/companion_fr.ts (hunk 3 above)
Comment (mpaperno) [inline companion/src/translations/companion_fr.ts]: They keyboard keys can/should certainly be translatable (to match local keyboards, if necessary).  They radio key labels should probably match the actual radio hardware (assuming that's not localized in most cases).

**Comment by LapinFou** [comment]
Updated keyboard shortcuts according @mpaperno comment.

File: companion/src/translations/companion_fr.ts (hunk 3 above)
Comment (bsongis) [inline companion/src/translations/companion_fr.ts]: Indeed but I don't think html should be translatable! We can change this later

Comment (mpaperno) [inline companion/src/translations/companion_fr.ts]: Perhaps, but the HTML in this case is part of the formatting which is going to depend on the size of the characters used (eg. some use font size +3, others +2, others none, etc).  This way the whole indicator string can be replaced if necessary (eg. with an icon) w/out being tied to a particular font size (or any other attributes included in the HTML).  Granted this possibly requires a bit more "comprehension" on the part of the translator (or at least testing), but OTOH it is a lot more flexible.

Comment (LapinFou) [inline companion/src/translations/companion_fr.ts]: Maybe, a `Developer comments` should be added to avoid any confusion?

**Pull request merged**